client/src/components/RatingsReviews/NewReview.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from 'react';
+import {
+  Grid,
+  TextField,
+  MenuItem,
+  Modal,
+  Button,
+  Typography,
+  Radio,
+  RadioGroup,
+  FormControl,
+  FormControlLabel,
+  FormLabel
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import Rating from '@material-ui/lab/Rating';
+import CharacteristicsRadio from './CharacteristicsRadio.jsx';
+
+const useStyles = makeStyles({
+  modal: {
+    position: 'absolute',
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+    width: 450,
+    'background-color': 'white',
```

Comment:
For React, you'd need to set `backgroundColor: 'white',` but hey... if it works, it works right?